<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CPU Saver control</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../Downloads/cpu-icon.png</normaloff>../../../../../Downloads/cpu-icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="exListWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>113</y>
      <width>201</width>
      <height>210</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="thrLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Threshold:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="thrSpinbox">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>10</y>
      <width>60</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="endTypeLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>47</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>End type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="endTypeComboBox">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>44</y>
      <width>86</width>
      <height>29</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="percentLabel">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>83</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Percent on SD</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="percentSpinBox">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>77</y>
      <width>39</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="exNameLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>9</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Excep cmd</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exLineEdit">
    <property name="geometry">
     <rect>
      <x>339</x>
      <y>4</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exEndLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>45</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ex end type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="exEndComboBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>43</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addExButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>76</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+ exception</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remExButton">
    <property name="geometry">
     <rect>
      <x>358</x>
      <y>76</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>- exception</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getcfgButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setcfgButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>165</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set config</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isRunningButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>232</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Is daemon running ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopdButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>277</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startdButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>277</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="auxLabel">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>82</y>
      <width>27</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>/10</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAsdasdad">
   <property name="text">
    <string>asdasdad</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
